core-bundle/src/Controller/AbstractController.php

```diff
@@ -38,7 +38,7 @@ public static function getSubscribedServices(): array
         $services['logger'] = '?'.LoggerInterface::class;
         $services['fos_http_cache.http.symfony_response_tagger'] = '?'.SymfonyResponseTagger::class;
         $services['contao.csrf.token_manager'] = ContaoCsrfTokenManager::class;
-        $services['contao.cache.entity_tags'] = EntityCacheTags::class;
+        $services['contao.cache.tag_manager'] = CacheTagManager::class;
```

Comment:
I'm not sure we are allowed to remove that service here. It would break controllers using `contao.cache.entity_tags` now, right? It should probably be left here but deprecated when called?

Reply:
I thought about this too. In theory, you should get those services injected and never use the container (also see https://github.com/contao/contao/pull/7780/files#diff-18e6b26f7de75e3f9cb17f471d6550302da796db90926ab075bfe55f8502edfd), but I agree there's a chance people used it. I don't know if you would get a deprecation everywhere because we keep it.

Reply:
It's fine to remove this. If you are using a service in your own code, you need to add it yourself. It's the same as relying on a composer package that was required by someone else. :slightly_smiling_face: